### DISCUSSION TITLE
cp950 encoding error

### DISCUSSION THREAD

**Discussion by alskdj987** [ansys/pyaedt#5205] [Q&A] [answered]
When I tried to execute the sample code, I encountered this problem. I would like to ask if I can only find all open in the source code of ansys and change the encoding='utf-8'

PyAEDT ERROR:     h3d.modeler.layers.add_layer(layer="GND", layer_type="signal", thickness="0", isnegative=True)
PyAEDT ERROR: 'cp950' codec can't decode byte 0xc3 in position 37185: illegal multibyte sequence on _read_materials
PyAEDT ERROR: Last Electronics Desktop Message - [error] 'cp950' codec can't decode byte 0xc3 in position 37185: illegal multibyte sequence on _read_materials

**Comment by Samuelopez-ansys** [comment]
Hi @alskdj987 , Please share for information. Which sample code?

What are you trying to do ?

**Comment by alskdj987** [comment]
I try to run 
pyaedt/examples/01-HFSS3DLayout/HFSS3DLayout_Via.py
I just want to test whether pyaedt can execute
environment:
python 3.11.9
pyaedt v0.10.2

**Comment by alskdj987** [comment]
I only execute the first half of the program

import ansys.aedt.core
import os

aedt_version = "2024.1"

non_graphical = True

h3d = ansys.aedt.core.Hfss3dLayout(version=aedt_version, new_desktop=True, non_graphical=non_graphical)
h3d["viatotrace"] = "5mm"
h3d["viatovia"] = "10mm"
h3d["w1"] = "1mm"
h3d["sp"] = "0.5mm"
h3d["len"] = "50mm"

h3d.modeler.layers.add_layer(layer="GND", layer_type="signal", thickness="0", isnegative=True)

**Comment by maxcapodi78** [comment, marked as answer]
Hello @alskdj987 I think this could be related to some material amat file you have saved in your personallib or userlib. I've modified the code as in #5211 to skip wrong or not readable files
